sdk/core/Azure.Core/tests/TestFramework/RecordedTestBase.cs

```diff
@@ -39,14 +39,15 @@ private string GetSessionFilePath(string name = null)
             return Path.Combine(TestContext.CurrentContext.TestDirectory, "SessionRecords", className, fileName);
         }
 
-        public TestRecording StartRecording(string name)
-        {
-            return new TestRecording(Mode, GetSessionFilePath(name), Sanitizer, Matcher);
-        }
-
         [SetUp]
         public virtual void StartTestRecording()
         {
+            TestContext.TestAdapter test = TestContext.CurrentContext.Test;
+            if (Mode != RecordedTestMode.Live &&
+                test.Properties.ContainsKey("SkipRecordings"))
+            {
+                throw new IgnoreException((string) test.Properties.Get("SkipRecordings"));
```

Comment:
Nit - add a comment like `// Only create test recordings for the latest version of the service`?